docker-compose.yml

```diff
@@ -307,6 +307,9 @@ services:
     ports:
       - "3100:3100"
     command: -config.file=/etc/loki/local-config.yaml
+    volumes:
+      - ./logging/loki-config.yaml:/etc/loki/local-config.yaml
+      - loki:/data/loki
```

Comment:
The path would be `/loki `only because the YAML config has the same for chunks config

Reply:
Changes implemented.